ibis/backends/impala/tests/snapshots/test_value_exprs/test_timestamp_extract_field/millisecond/out.sql

```diff
@@ -1 +1 @@
-extract(`i`, 'millisecond')
\ No newline at end of file
+extract(`i`, 'millisecond') % 1000
```

Comment:
is this correct?

Reply:
Yeah, this was surprising to me too. The behavior of `extract` for micros and millis changed in Impala.

Reply:
weeeeird.  ok!